docs/contributing/javaagent-jar-components.md

```diff
@@ -82,7 +82,7 @@ module
 Available in the bootstrap class loader:
 
 - `io/opentelemetry/javaagent/bootstrap/` - contains the `javaagent-bootstrap` module
-- `io/opentelemetry/instrumentation/auto/api/` - contains the `auto-api` module
+- `io/opentelemetry/javaagent/api/` - contains the `javaagent-api` module
```

Comment:
Currently it's `io/opentelemetry/javaagent/instrumentation/api/`

Reply:
thanks! fixed